com.zenorogue.HyperRogue.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>com.zenorogue.HyperRogue</id>
+  <metadata_license>CC-BY-3.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <content_rating type="oars-1.0" />
```

Comment:
This is wrong, there is at least some violence in the game if screenshots are accurate.